analyser_merge_milestone_FR_metropole.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Frédéric Rodrigo 2014-2020                                 ##
```

Comment:
Your name here.

Reply:
Please fix this.

Reply:
i don't know how ...

Reply:
Just like you make other changes.

Reply:
there is not commit button or ...

Reply:
I does not know how you make your other changes.

You can make a commit locally, and push the commit (the normal way to do it).

You can also make a commit on github. Using the '...' menu on file, then edit.

Or in comment using suggestion (like I did for you) https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review